### PR TITLE
add back installer instructions for docs

### PR DESCRIPTION
**Pull request by evilnick**
## Done

Adds back the instructions for Windows and Mac installers

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

### CODE REVIEW

File: docs/install-alternatives.md [hunk 1]
```diff
@@ -218,18 +286,12 @@ For an example, see this [answer on askubuntu][askubuntu].
 
 <!-- LINKS -->
 
+[uninstall]: /docs/uninstall-alternatives
```
Comment (anthonydillon) [inline docs/install-alternatives.md:289]: This link 404's

File: docs/install-alternatives.md [hunk 2]
```diff
@@ -15,28 +15,66 @@ notes below.
...
+    hypervisor feature in Windows 10. To do so, please follow the
+    instructions on the [MicroSoft website][hyper-v].
 
-For now, the best way to run MicroK8s on Windows is with virtualisation.
-MicroK8s will install without problems on Ubuntu running on number of different
-VMs, including [VirtualBox](https://www.virtualbox.org/).
+1.  **Download the installer**
+
+    The Windows installer is available on the MicroK8s GitHub page.
+    [Download it here](https://github.com/ubuntu/microk8s/releases/download/installer-v1.0.0/microk8s-installer.exe)
```
Comment (anthonydillon) [inline docs/install-alternatives.md:36]: This is a pre-release, do you want to hold off until the release officially happens?

File: docs/install-alternatives.md (hunk 1 above)
Comment (evilnick) [reply docs/install-alternatives.md:289]: it links to a page which was landing in another branch. I will refresh

File: docs/install-alternatives.md (hunk 2 above)
Comment (evilnick) [reply docs/install-alternatives.md:36]: thanks for the heads up, I'll sit on it for now

**Comment by anthonydillon** [comment]
@evilnick is this one ready to go yet?

File: docs/install-alternatives.md (hunk 2 above)
Comment (anthonydillon) [reply docs/install-alternatives.md:36]: I believe this has been released now?

**Comment by anthonydillon** [comment]
@evilnick can this merge or be closed?

**Pull request closed without merging**